src/scss/_header.scss

```diff
@@ -147,3 +147,38 @@ header {
     }
   }
 }
+
+//* Header sign
+
+.header-sign {
+  font-family: 'Sue Ellen Francisco', cursive;
+  font-size: 2.2rem;
+  letter-spacing: 0.1em;
+  transform: rotate(-7.65deg);
+  margin-top: 4rem;
+  width: 30rem;
+}
+
+.header-sign:before {
```

Comment:
Pseudo elements should use double colon --> `.header-sign::before`. Using single is an outdated syntax.

Reply:
Fixed